DTS-STN/c19-benefits-node
# Change/privacy tand c link update

# Description

Updated the links in the footer to reflect the new location of privacy agreement and terms and conditions.


## Test Instructions

1. Click the links in the footer, verify they take you to the proper location. Proper link locations can be found [here](https://taiga.dts-stn.com/project/demoore-covid-benefits-finder/us/78?kanban-status=106) 

## Checklist
